src/types/faq.tsx

```diff
@@ -294,7 +294,10 @@ export const faqQuestionsAndAnswers: TopicQuestionAndAnswers[] = [
         answer: (
           <span>
             Check out our{' '}
-            <A newWindow={false} href="/governance/overview">
+            <A
+              newWindow={false}
+              href="https://developer.thenewboston.com/whitepaper/principal-entities#principal-entities-governance"
```

Comment:
hey, instead of hardcoding the complete URL can you use the constants we've made in `constants/routes`?